pom.xml

```diff
@@ -109,6 +110,13 @@
           <version>${grpc.version}</version>
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-tcnative-boringssl-static</artifactId>
+          <version>${netty-tcnative.version}</version>
+          <classifier>${os.detected.classifier}</classifier>
+          <scope>test</scope>
```

Comment:
Suppose that I export jetcd as jetcd.jar. If the user wants to use jetcd.jar, then the user also needs include a ` netty-tcnative` pkg in order to build the code with jetcd.jar properly? since jetcd needs to be dynamically linked to a `netty-tcnative` pkg. Is my understand of `dynamic linked` in for java jar correct?